polyfills/Promise/config.json

```diff
@@ -14,7 +14,7 @@
 		"bb": "* - 10",
 		"chrome": "* - 31",
 		"firefox": "6 - 28",
-		"ie": "8 - *",
+		"ie": "< 12",
```

Comment:
You're extending this polyfill to IE7 here.  I don't believe the polyfill works in IE7.